defi/l2/utils.ts

```diff
@@ -380,9 +273,8 @@ export async function fetchSupplies(
   }
 }
 export async function fetchBridgeTokenList(chain: Chain): Promise<Address[]> {
-  const j = Object.keys(incomingAssets).indexOf(chain);
   try {
-    const tokens: Address[] = j == -1 ? [] : await Object.values(incomingAssets)[j]();
+    const tokens: Address[] = incomingAssets[chain as keyof typeof incomingAssets] ? await incomingAssets[chain as keyof typeof incomingAssets]() : []
```

Comment:
one of the reasons why I hate typescript, it makes to code unreadable, no need to change, personally I prefer `any` over this